WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -30,39 +29,47 @@ private struct AccountSettingsController: SettingsController {
     // MARK: - Initialization
 
     let service: AccountSettingsService
+    var settings: AccountSettings? {
+        didSet {
+            NSNotificationCenter.defaultCenter().postNotificationName(ImmuTableViewController.controllerChangedNotification, object: nil)
```

Comment:
Since `Settings` represents the model, and not the *controller*, maybe `modelChangedNotification` instead?